### DISCUSSION TITLE
欢迎来到情侣小程序 讨论大厅！

### DISCUSSION THREAD

**Discussion by UxxHans** [UxxHans/Rainbow-Cats-Personal-WeChat-MiniProgram#26] [Announcements]
## 👋 Welcome! 欢迎来到讨论区! 这里可以讨论项目!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about. 可以问问题哦！
  * Share ideas. 分享点子！
  * Engage with other community members. 和其他社区小伙伴聊天！
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪. 一起共建美好社区吧！💪

**Comment by byDouDou** [comment]
请问这个miniprogram/envList.js是在哪里打开呀

**Reply by UxxHans** [reply]
![image](https://user-images.githubusercontent.com/90906577/181766438-4cf35daf-f2fb-4323-ad28-2e1f3290d555.png)
在这里

**Reply by UxxHans** [reply]
图片可能无法加载 文件位置就是这个路径miniprogram/envlist.js

**Reply by byDouDou** [reply]
请问这个页面是怎么进入的呀 是在手机版小程序开发里吗

**Reply by UxxHans** [reply]
请先把所有文件从github仓库上下载下来

**Reply by byDouDou** [reply]
您好我已经把所有文件下载下来了然后请问如何把cloudfunction下的文件夹导入到云函数呢

**Reply by mmdoctor** [reply]
其余功能都正常，唯独无法发布商品，请问如何解决呢

**Comment by Nebulas123** [comment]
使用逻辑里能不能增加悬赏呢，例如女朋友想要我帮她买零食，应该是她来发布采购任务，我完成之后获得积分，目前的逻辑只能发布者获得积分呀

**Reply by Nebulas123** [reply]
哦我注意到啦#21

**Comment by BrotherHai** [comment]
大佬有没有搞一个uniapp版本的打算呀，微信小程序被封了好无奈

**Reply by UxxHans** [reply]
想做个Unity版本的

**Comment by SSFeb-1** [comment]
呜呜呜救救孩子 有些地方mac系统好像对不上  想问问运行cloudfunctions/Install-WX-Server-SDK.bat要怎么运行呀？（爆哭）

**Reply by Nebulas123** [reply]
batch是windows的脚本语言，mac当然用不了呀

**Reply by SSFeb-1** [reply]
> batch是windows的脚本语言，mac当然用不了呀

嘿嘿 好滴 我试了您写在后面的（在命令行输入 npm install --save wx-server-sdk@latest）成功了耶 ！！  感谢大佬无私分享 已经用上啦 嘿嘿

**Comment by Tipenso** [comment]
小程序发布测试版后，制定任务无法保存，大大怎么解决呀

**Reply by sostoptalk** [reply]
我也是无法保存

**Comment by WillWang1113** [comment]
你们一般都上架什么商品给对象呀hhh来取经了

**Reply by Charry-039** [reply]
捶背券

**Comment by byDouDou** [comment]
想请问一下云函数那一步如何上传呀

**Comment by SuWeiYi100** [comment]
显示credit of undefined 怎么啊

**Reply by dreamer-295** [reply]
我也是 你解决了吗

**Comment by yyyyagagggas** [comment]
大佬这边小程序中制定的任务无法保存是怎么回事啊

**Comment by admin121253a** [comment]
完成任务没有积分

**Reply by haojingkexudu** [reply]
没有修改数据库吧

**Comment by chenningshih** [comment]
![image](https://github.com/UxxHans/Rainbow-Cats-Personal-WeChat-MiniProgram/assets/141300411/aa3b3bb9-4b51-4957-ab27-7317fdc80b01)这个路径是什么呀

**Reply by haojingkexudu** [reply]
应该是首页面吧

**Reply by chenningshih** [reply]
miniprogram/pages/MainPage/index.wxml 我配置这个路径也不对哎

**Reply by haojingkexudu** [reply]
![image](https://github.com/UxxHans/Rainbow-Cats-Personal-WeChat-MiniProgram/assets/49332698/e1078ecf-83ca-49ed-9fea-b9ec2a397a4d)
这是我的路径可以供你参考

**Reply by chenningshih** [reply]
可以了 很nice

**Comment by sostoptalk** [comment]
在手机上添加不了任务和商品，请问怎么解决呀

**Reply by sostoptalk** [reply]
点保存之后没反应

**Comment by huyueliang1** [comment]
完成后没有积分 数据库的问题吗 我记得修改了啊

**Reply by mmdoctor** [reply]
openid没更改，代码里的和数据库都要改

**Comment by mmdoctor** [comment]
在UserList中需要添加两个默认记录，先测试下双方是否都能发布和完成任务，对方刷新后任务是否增加和完成。如果都能，那就是计算积分部分代码出问题了

________________________________
发件人: 图八 ***@***.***>
发送时间: 2025年2月10日 21:25
收件人: UxxHans/Rainbow-Cats-Personal-WeChat-MiniProgram ***@***.***>
抄送: Mr.R ***@***.***>; Comment ***@***.***>
主题: Re: [UxxHans/Rainbow-Cats-Personal-WeChat-MiniProgram] 欢迎来到情侣小程序 讨论大厅！ (Discussion #26)


我的完成之后，只有数据库userList第一个用户完成之后才会有积分变化，已经确认了openid是没错的，求大神指教！
[Uploading 屏幕截图 2025-02-10 212110.png…]

―
